jobs/JREGIONAL_NONVARCLD

```diff
@@ -64,7 +64,7 @@ if [ ${CYCLE_TYPE} == "spinup" ]; then
 else
   analworkname="nonvar_cldanl"
 fi
-if [ ${ENS_TYPE} == "MEAN" ]; then
+if [ ${RRFSENS_TYPE} == "MEAN" ]; then
```

Comment:
A general question, why we would like to change "ENS_TYPE" to "RRFSENS_TYPE"?  Is there a problem to continue using "ENS_TYPE"?

Reply:
There is another "ENS_TYPE" in the GSI analysis script, which was used for the ensemble for the hybrid analysis. Therefore I have to define it in another name.

Reply:
Thanks. However, "rrfsens_type" and "ens_type" still look similar to me and I think it will also bring confusions to others.  Would it be better to change "RRFSENS_TYPE" to "MEAN_OR_MEM", "MEM_TYPE" or "MEMBER_TYPE"? Just my 2 cents.

Reply:
We may want to change ENS_TYPE in GSI to ENS_TYPE4HYB and let real ensemble use ENS_TYPE.

Reply:
> We may want to change ENS_TYPE in GSI to ENS_TYPE4HYB and let real ensemble use ENS_TYPE.


ENS_TYPE4HYB and RRFSENS_TYPE still look confusion to me.  Another thought based on you idea is that we may simply change "ens_type" in GSI to "suffix" and let Chunhua keep using "ENS_TYPE" (or change to "MEM_TYPE").

Reply:
Now I think "MEM_TYPE" works better for this "MEAN" or "MEMBER" choice, than "ENS_TYPE".
What do we want to use for the "ENS_TYPE" in GSI run script?

Reply:
For GSI, I suggest to use "ftype"  (i.e file_type)